Adafruit_MLX90614.h

```diff
@@ -62,6 +57,8 @@ class Adafruit_MLX90614 {
   void writeEmissivity(double emissivity);
 
 private:
+  Adafruit_I2CDevice *i2c_dev = NULL; ///< Pointer to I2C bus interface
+  uint8_t buffer[4];
```

Comment:
this isnt really something we do in C, the way we have it in circuitpy. you should have the buffer be allocated on the stack in the function thats using it, not trying to be clever and reusing the same buffer.